tools/tscdocgen/main.go

```diff
@@ -268,6 +268,14 @@ func (e *emitter) augmentNode(node *typeDocNode, parent *typeDocNode) {
 	node.Label = createLabel(node, parent)
 	node.CodeDetails = createCodeDetails(node)
 	node.URLPath = getURLPath(node, parent)
+	if k8s && node.Comment.ShortText != "" &&
+		strings.Contains(node.Comment.ShortText, "<") &&
+		strings.Contains(node.Comment.ShortText, ">") {
+		// To avoid double-encoding strings that are already encoded, we make a targeted replacement
+		// of the < and > characters alone.
+		node.Comment.ShortText = strings.ReplaceAll(node.Comment.ShortText, "<", "&lt;")
+		node.Comment.ShortText = strings.ReplaceAll(node.Comment.ShortText, ">", "&gt;")
```

Comment:
We still don't need the `node.Comment.ShortText != ""` and `string.Contains` checks, right?

So it could be simply:

```go
if k8s {
	// To avoid double-encoding strings that are already encoded, we make a targeted replacement
	// of the < and > characters alone.
	node.Comment.ShortText = strings.ReplaceAll(node.Comment.ShortText, "<", "&lt;")
	node.Comment.ShortText = strings.ReplaceAll(node.Comment.ShortText, ">", "&gt;")
}
```

Might also be worth adding a little more detail to the comment about why we only do this for Kubernetes (so that `<namespace>/<name>`, which shows up a lot, is properly escaped), but don't for other providers (because there are cases for other providers we do want`<` and `>` to be emitted unescaped).

Reply:
I'll make the updates.

Reply:
However, I'll leave the explicit check for `<` and `>` even though it is restricted to k8s now. Just to avoid any other unintended changes.

Reply:
Oh, I see, the checks are ensuring both `<` and `>` are in the string before doing the replacements. Got it.